independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Beans.java

```diff
@@ -261,12 +252,14 @@ static BeanInfo createProducerMethod(MethodInfo producerMethod, BeanInfo declari
                 alternativePriority = declaringBean.getAlternativePriority();
             }
             alternativePriority = initAlternativePriority(producerMethod, alternativePriority, stereotypes, beanDeployment);
-            // after all attempts, priority is still null
             if (alternativePriority == null) {
                 // after all attempts, priority is still null, bean will be ignored
-                LOGGER.infof(
-                        "Ignoring producer method %s - declared as an @Alternative but not selected by @Priority, @AlternativePriority or quarkus.arc.selected-alternatives",
-                        declaringBean.getTarget().get().asClass().name() + "#" + producerMethod.name());
+                String message = "Ignoring producer method %s - declared as an @Alternative but not selected by @AlternativePriority or quarkus.arc.selected-alternatives";
```

Comment:
Why remove the "`@Priority`" part? :-)

Reply:
See https://github.com/quarkusio/quarkus/pull/13427#discussion_r528589032